packages/amplify-e2e-tests/package.json

```diff
@@ -117,7 +117,10 @@
     "moduleNameMapper": {
       "axios": "axios/dist/node/axios.cjs",
       "^csv-parse/sync": "<rootDir>/../../node_modules/csv-parse/dist/cjs/sync.cjs"
-    }
+    },
+    "coveragePathIgnorePatterns": [
+      "/__tests__/"
```

Comment:
Probably remove L77-79 above, and update these patterns with node modules & `*.d.ts` files

Reply:
I'm going to remove the `*.d.ts` pattern because it won't be used. `*.d.ts` files will be in `lib` which won't have coverage collected.